scoresheets/Restaurant.tex

```diff
@@ -3,19 +3,21 @@
 	\scoreitem[2]{100}{Detect calling or waving customer}
 	\scoreitem[2]{100}{Reach a customer's table without prior guidance/training}
 
-	\scoreitem[2]{300}{Take an order.}
-	\scoreitem[2]{300}{Serve an order.}
+	\scoreitem[2]{200}{Understand and confirm the order received to the customer}
+	\scoreitem[2]{100}{Communicate the order to the barman}
+
+	\scoreitem[2]{100}{Return to the customer table with the order}
+	\scoreitem[2]{200}{Serve the order byplacing the items on the table}
```

Comment:
```suggestion
	\scoreitem[2]{200}{Serve the order to the customer}
```
Lets keep this open (no "by placing"). Makes it easier to understand and apply the `handover` penalties